README.md

```diff
@@ -133,6 +133,29 @@ K9s is available on Linux, macOS and Windows platforms.
 ## Building From Source
 
  K9s is currently using go v1.14 or above. In order to build K9s from source you must:
+ installing go language
+
+ 
+ ```bash
+wget https://golang.org/dl/goX.Y.Z.linux-amd64.tar.gz
+
+sudo tar -xvf go1.21.0.linux-amd64.tar.gz -C /usr/local
```

Comment:
Might as well continue with the X.Y.Z theme...

Reply:
Yah I agree with you and I removed it